Cleaner pokestop tag

gsisso opened the pull request.

gsplash committed:
Update TagPokestop.java

kylestev commented:
@gsplash awesome. just waiting on Codacy and Travis before merging!

kylestev commented:
@gsplash Have you joined Slack? The information is in the README.

gsplash committed:
Update TagPokestop.java

kylestev commented:
Thanks for fixing that up.

kylestev commented:
Uh oh. The build failed @gsplash

kylestev commented:
You'll want to take a look at the push I just made. In particular SwipeX/PokeMate@b8ea1d699bb80d994a308e7097d17d1eb2dc840a

gsplash committed:
Update TagPokestop.java

kylestev commented on src/main/java/dekk/pw/pokemate/tasks/TagPokestop.java:
```diff
@@ -52,66 +55,16 @@ private String resultMessage(final PokestopLootResult result) {
+                }
+                //build the rest of the string
+                for (Map.Entry<Integer, Integer> item : receivedItems.entrySet()) {
+                    retstr += " - " + StringConverter.convertItem(ItemId.valueOf(item.getKey()).name()) + "(x" + item.getValue() + ")";
```
You'll want to pull and change this to `StringConverter#titleCase`

gsplash committed:
Update TagPokestop.java

gsplash committed:
Update TagPokestop.java

gsplash committed:
Update TagPokestop.java

kylestev commented:
@gsplash the issue is that you have not pulled the changes from this repository. I rebased it and merged it for you. See SwipeX/PokeMate@87a1fcf26163aa326eca2af03b6fe90752500295

gsisso commented:
@kylestev I already changed the string converter in few previous commits ,it just failed because  `StringConverter.titleCase(ItemId.valueOf(item.getKey()).name())` 
should be `(StringConverter.titleCase(ItemId.valueOf(item.getKey()).name()))`
dunno why it cant build it this way ,my compiler does it just fine. anyway I fixed it and you might want to pull the latest commit as well

kylestev commented:
But you did not have the change for titleCase pushed to github, thus the failing check -- it was not because of the parenthesis around StringConverter and the method call. I tried compiling your branch locally and the build failed both prior to and after you added the parenthesis.

![](https://i.imgur.com/7VLStlM.png)

You can see the branch diff here https://github.com/gsplash/PokeMate/compare/master...SwipeX:master
